File: manual_fixtures/integration_guide/instantiate.js
var Client = require('service-registry-client').Client;

var username = ''; // your username here
var key = ''; // your API key here
var service_registry_url = 'https://dfw.registry.api.rackspace.com/v1.0/';

var client = new Client(username, key, 'us', {'url': service_registry_url});

File: manual_fixtures/integration_guide/instantiate.py
from txServiceRegistry.client import Client
from twisted.internet import reactor

RACKSPACE_USERNAME = '' # your username here
RACKSPACE_KEY = '' # your API key here
SERVICE_REGISTRY_URL = 'https://dfw.registry.api.rackspace.com/v1.0/'

client = Client(username=RACKSPACE_USERNAME,
                apiKey=RACKSPACE_KEY,
                baseUrl=SERVICE_REGISTRY_URL,
                region='us')
